lib/libutee/tee_api_objects.c

```diff
@@ -164,18 +154,18 @@ TEE_Result TEE_GetObjectValueAttribute(TEE_ObjectHandle object,
 
 	res = utee_cryp_obj_get_info((uint32_t)object, &info);
 	if (res != TEE_SUCCESS)
-		TEE_Panic(0);
-
-	if ((info.handleFlags & TEE_HANDLE_FLAG_INITIALIZED) == 0)
-		TEE_Panic(0);
+		goto exit;
```

Comment:
The check *"Panic Reasons: The object is not initialized."* has been removed. Is it on purpose?